app/src/forms/form/exportService.js

```diff
@@ -81,6 +81,17 @@ const service = {
   },
 
   _formatData: async (exportFormat, exportType, form, data = {}) => {
+
+    // if form doesnt have status updates enabled in the form settings
+    if(!form.enableStatusUpdates) {
+      // remove status related attributes from data set
+      data.forEach(function(obj){
+        delete obj.status;
+        delete  obj.assignee;
+        delete  obj.assigneeEmail;
+      });
+    }
+
```

Comment:
this could be a good place to do any general formatting. for example, renaming meta columns fullName to 'Full Name' or 'form.fullName'